release.nix

```diff
@@ -2,7 +2,7 @@
 , scrubJobs ? true
 , haskell-nix ? { outPath = ./.; rev = "abcdef"; }
 , nixpkgsArgs ? {}
-, ifdInputsOnly ? false
+, ifdInputsOnly ? true
```

Comment:
What's this?

Reply:
A hack to get good feedback from hydra when changing stuff in haskell.nix.  Setting it to `true` disables the tests that uses an IFD and instead just builds the inputs (nix files).  That way you get good feedback on what might otherwise be built in the `eval` phase.  Once those tests are all working you can set it back to `false` and get all the test results.

I was going to put ifdInputsOnly in a PR by itself, but I needed it here because we changed `comp-builder.nix` enough that `ghc` is rebuilt and if that happens during `eval` hydra gives no feedback.

Ideally we could get hydra to automatically run `release.nix` with it first set to `true` then set to `false` only once that succeeds.

I'll write some comments and docs and we can included it in this PR or I can separate it out if you would prefer.